Tests/smokeTest.js

```diff
@@ -0,0 +1,41 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
```

Comment:
Try to be consistent in pages naming, for example use loginPage instead of loginpage here (only first letter should be lowercase). For fashionCategory also add "Page" part.

Reply:
I fixed this.